keyboards/marcopad/readme.md

```diff
@@ -0,0 +1,27 @@
+# Marcopad
+
+![macropad](imgur.com image replace me!)
+
+*A compact 3x3 keys macropad built with RP-2040 Zero*
+
+* Keyboard Maintainer: [Marco Pelegrini](https://pelegrini.ca/marcopad)
+* Hardware Supported: [Macropad PCB](https://pelegrini.ca/marcopad/hardware)
+* Hardware Availability: [Macropad](https://pelegrini.ca/marcopad)
```

Comment:
These links are all dead

Reply:
I'm publishing the site at the moment, chicken and egg situation. Will close this when done